modules/cli/src/main/java/org/apache/ignite/cli/builtins/config/ConfigurationClient.java

```diff
@@ -81,7 +81,7 @@ public String get(
     ) {
         var req = HttpRequest
                 .newBuilder()
-                .header("Content-Type", "application/json");
+                .header("Content-Type", "text/plain");
```

Comment:
I've changed the Content-Type from applicaiton/json to text/plain because the HOCON configuration, in fact, is represented as a text (string) but not as a json object.

Reply:
Makes sense, can you please leave a comment?